lib/cocoapods/installer/analyzer.rb

```diff
@@ -643,16 +641,18 @@ def dependencies_for_specs(specs, platform, all_specs)
       # @return [PodTarget]
       #
       def generate_pod_target(target_definitions, target_inspections, specs, scope_suffix: nil)
+        target_requires_64_bit = target_definitions.any? { |td| requires_64_bit_archs?(td.platform) }
```

Comment:
should this be `all?` This is why its a good idea to add a spec with more than one target.

Reply:
yes I think you're right, will look into this and add another test